### DISCUSSION TITLE
Lazy Loading Support?

### DISCUSSION THREAD

**Discussion by rollsrobby** [microsoft/fluentui#28270] [Q&A] [answered]
I'm using the latest next.js and flunet ui 9 where I built a wizard/stepper form, lazy loading components based on the selection on the current screen. Everything was working fine in debug but as soon as I deployed I got a `using disposed tabster` error. 

So this is a **general question**. Is [lazy loading components](https://react.dev/reference/react/lazy) supported when using fluentui?

**Comment by layershifter** [comment, marked as answer]
@rollsrobby this looks like a bug, please create an issue with minimal reproduction case, we will check it 👍

**Reply by rollsrobby** [reply]
sorry for the late answer, I was unable to create a minimal reproduction case that had the issue. In the meantime we refactored the code and it seems to work now. So I guess it was something we had on our end that caused the issue. ✌️
